src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,15 @@
+status = info
```

Comment:
pom.xml has to be changed to remove the properties file being included from RMB

Reply:
Updated. One note. To filter log4j2 props provided by RMB, I used following filter
```xml
<artifact>org.folio:domain-models*</artifact>
```
instead of 
```xml
<artifact>org.folio:raml-module-builder</artifact>
```
Based on info message in console:
```
[INFO] No artifact matching filter org.folio:raml-module-builder
```